stdlib/area/position.lua

```diff
@@ -201,4 +201,21 @@ function Position.tostring(pos)
     end
 end
 
+--- Returns the opposite direction - Adapted from Factorio util.lua
+-- @param direction the defines.direction to get the opposite of
+-- @return the opposite direction
+function Position.opposite_direction(direction)
+    local opposites = {
```

Comment:
It is still a loss, and I can imagine *many* cases this could be called in a tight loop, so even a tiny loss builds up.  :-)